suites/dream-challenge/synapaseLoginTest.js

```diff
@@ -0,0 +1,84 @@
+Feature('Synapse Login')
+
+const user = require('../../utils/user.js');
+const chai = require('chai');
+const {interactive, ifInteractive} = require('../../utils/interactive.js');
+const expect = chai.expect;
+const hostname = process.env.HOSTNAME;
+const profile = process.env.NAMESPACE;
+
+Scenario('Login with Google / Microsoft @manual', ifInteractive(
+    async(I) => {
+        const result = await interactive (`
+            1. Go to https://qa-brain.planx-pla.net
+            2. Check Dictionary button and check if graphic representation of data model is visible
+            3. Also check other button on the top menu bar such as Exploration, Files and Profile. 
```

Comment:
we are going to remove the `Files` nav bar button soon since user can get to the files page via `Exploration` -> `File`